src/extension.ts

```diff
@@ -422,7 +439,12 @@ function createNewIntegration(context: vscode.Uri): Promise<boolean> {
 				console.error(error); 
 			});
 
-		let commandString = 'kamel run --dev "' + absoluteRoot + '"';
+		let commandStringRoot = 'kamel run';
+		if (useDevFlag) {
+			commandStringRoot = commandStringRoot + ' --dev ';
```

Comment:
what about writing it commandStringRoot += ' --dev '; ?
I prefer this one as it is shorter. I find it more easy to read also, i don' thave to check that the first argument is written exactly the same than the assignment.